tornjak-frontend/src/components/spiffe-helper.js

```diff
@@ -179,21 +180,62 @@ class SpiffeHelper extends Component {
     return "";
   }
 
-  // numberEntries takes in spiffe id of an agent and list of entries
-  // returns number of entries in an agent
-  numberEntries(spiffeid, globalEntries, globalAgents) {
-    if (typeof globalEntries !== 'undefined') {
-      var entriesList = this.getChildEntries(spiffeid, globalAgents, globalEntries);
-      if (typeof entriesList === 'undefined') {
-        return NaN
-      } else {
-        return entriesList.length
-      }
-    } else {
-      return NaN
+  // numberEntriesOfAgent takes in an agent and list of entries
+  // returns number of entries in the agent
+  numberEntriesOfAgent(agent, globalEntries) {
+    if (globalEntries === undefined) {
```

Comment:
same as above:
```
if (!globalEntries) {
  return;
}
```

Reply:
File not of concern on this PR. Will be addressed in the next consecutive PR.